src/lib/storage/proposal/devicegraph_generator.rb

```diff
@@ -54,12 +54,12 @@ def devicegraph(volumes, initial_graph: ::Storage::Devicegraph.probed)
           graph
         end
 
-      protected
+        protected
 
         attr_writer :got_desired_space
 
         def got_desired_space?
-          !!@got_desired_space
+          @got_desired_space
```

Comment:
Then we should add another exception to the general YaST team rubocop configuration. Another setting we should not override locally.